content/en/docs/instrumentation/php/sdk.md

```diff
@@ -0,0 +1,100 @@
+---
+title: SDK
+weight: 8
+---
+
+The OpenTelemetry SDK provides a working implementation of the API, and can be
+set up and configured in a number of ways.
+
+## Manual setup
+
+When manually setting up an SDK, you have full control.
+
+```php
+<?php
+$exporter = new InMemoryExporter();
+$meterProvider = new NoopMeterProvider();
+$tracerProvider =  new TracerProvider(
+    new BatchSpanProcessor(
+        $exporter,
+        ClockFactory::getDefault(),
+        2048, //max queue size
+        5000, //export timeout
+        1024, //max batch size
+        true, //autoflush
+        $meterProvider
+    )
+);
+```
+
+## SDK Builder
+
+The SDK builder provides a fluent interface to configure parts of the SDK.
```

Comment:
fluent --> flexible?

Reply:
I might just change it to "provides an interface", but here I was referring to the fluent interface pattern (aka method chaining). That's probably not an important detail, though.

Reply:
Oh, cool! I know of method chaining, but wasn't aware of the use of the term "fluent" in that way. I'd say it's ok to leave as is. Thanks!